internal/version/version.go

```diff
@@ -23,6 +23,8 @@ import (
 // Version is the version of the build.
 const Version = "1.30.0"
 
+var ReleaseMinorVersions = []string{"1.29", "1.28", "1.27"}
```

Comment:
Let's add a doc comment 
```suggestion
// ReleaseMinorVersions are the currently supported minor versions.
var ReleaseMinorVersions = []string{"1.29", "1.28", "1.27"}
```